app/modules/site_settings/parameters.py

```diff
@@ -9,9 +9,9 @@
 from . import schemas
 
 
-class CreateSiteSettingParameters(Parameters, schemas.DetailedSiteSettingSchema):
-    class Meta(schemas.DetailedSiteSettingSchema.Meta):
-        fields = schemas.DetailedSiteSettingSchema.Meta.fields + (
+class CreateSiteSettingFileParameters(Parameters, schemas.DetailedSiteSettingFileSchema):
```

Comment:
What was site settings API will become a file only API, names changed to reflect this